frontend/src/entities/configuration/allowed-instance-types-configuration.tsx

```diff
@@ -0,0 +1,120 @@
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    ExpandableSection,
+    Header
+} from '@cloudscape-design/components';
+import React, { useMemo } from 'react';
+import { InstanceTypeMultiSelector } from '../../shared/metadata/instance-type-dropdown';
+
+export type AllowedInstanceTypesConfigurationProps = {
+    setFields: (any) => void;
+    expandedSections: {object};
+    setExpandedSections: (any) => void;
+    enabledNotebookInstanceTypes: string[];
+    enabledTrainingInstanceTypes;
+    enabledTransformInstanceTypes;
+    enabledEndpointInstanceTypes;
+};
+
+export function AllowedInstanceTypesConfiguration (props: AllowedInstanceTypesConfigurationProps) {
+    // Initialize the notebook instance type selectors with the currently selected options
+    const selectedNotebookInstanceOptions = useMemo(() => props.enabledNotebookInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledNotebookInstanceTypes]);
+
+    // Initialize the training job instance type selectors with the currently selected options
+    const selectedTrainingJobInstanceOptions = useMemo(() => props.enabledTrainingInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledTrainingInstanceTypes]);
+
+    // Initialize the transform job instance type selectors with the currently selected options
+    const selectedTransformJobInstanceOptions = useMemo(() => props.enabledTransformInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledTransformInstanceTypes]);
+
+    // Initialize the endpoint instance type selectors with the currently selected options
+    const selectedEndpointInstanceOptions = useMemo(() => props.enabledEndpointInstanceTypes.map((instance) => {
+        return {
+            value: instance,
+            label: instance,
+        };
+    }), [props.enabledEndpointInstanceTypes]);
+
+    return (
+        <>
+            <Container
+                header={
+                    <Header variant='h2'
+                        description='Modify the Amazon EC2 instance types that you wish to make available to users within the following resources: Notebook instance, Training and HPO jobs, Transform jobs, and Endpoints.'
+                    >
+                        Allowed Instance Types
+                    </Header>
+                }
+            >
+                <ExpandableSection headerText='Notebook instances' variant='default' expanded={props.expandedSections.notebookInstances} onChange={({ detail }) =>
```

Comment:
Do these still make sense to have as expandable sections? Expanding and contracting only saves us the space of an input field which at that point, might as well always have it open.

Reply:
Some of these can be long lists, I can see the value in these being expanding. If we want to remove them for containers we can make a new task/PR to address as this wasn't called out in the initial task.